lte/gateway/python/magma/pipelined/app/gy.py

```diff
@@ -233,8 +236,12 @@ def _install_redirect_flow(self, imsi, ip_addr, rule):
             rule_version=rule_version,
             priority=priority)
         try:
-            self._redirect_manager.setup_cwf_redirect(
-                self._datapath, self.loop, redirect_request)
+            if self._setup_type == 'CWF':
+                self._redirect_manager.setup_cwf_redirect(
+                    self._datapath, self.loop, redirect_request)
+            else:
+                self._redirect_manager.handle_redirection(
```

Comment:
can we rename it to setup_lte_redirection() to be consistent since both functions install redirect flows.